src/elements/content-preview/preview-header/FileInfo.js

```diff
@@ -9,13 +9,17 @@ import FileIcon from '../../../icons/file-icon/FileIcon';
 import './Header.scss';
 
 type Props = {
-    item: BoxItem | BoxItemVersion,
+    item: ?BoxItem | ?BoxItemVersion,
 };
 
 const FileInfo = ({ item }: Props) => (
     <div className="bcpr-name">
-        {item ? <FileIcon dimension={24} extension={item.extension} /> : null}
-        <span>{item.name}</span>
+        {item && (
+            <React.Fragment>
+                <FileIcon dimension={24} extension={item.extension} />
+                <span> {item.name} </span>
```

Comment:
Are the spaces inside the span needed? If they're being used for spacing, it's probably better to do it via CSS.